app/src/main/java/com/asma/tasky/feature_management/domain/AgendaItem.kt

```diff
@@ -1,53 +1,38 @@
 package com.asma.tasky.feature_management.domain
 
-import java.util.*
+import androidx.room.Entity
+import androidx.room.PrimaryKey
 
```

Comment:
Here in this file, you couple domain and data layers which isn't allowed. Domain should be fully isolated, so that even if your data layer changes, your domain and presentation layer won't.
But since you store your Room entities in here, your domain layer knows something about data specific implementations. And if you need to change your schema, you'll have to do it here instead of in data.